jupyterlab_h5web/__init__.py

```diff
@@ -1,5 +1,18 @@
+import json
+from pathlib import Path
+
+from ._version import __version__
 from .widget import H5Web
 
+HERE = Path(__file__).parent.resolve()
+
+with (HERE / "labextension" / "package.json").open() as fid:
+    data = json.load(fid)
+
+
+def _jupyter_labextension_paths():
+    return [{"src": "labextension", "dest": data["name"]}]
```

Comment:
Can't this be all in the function?
That would avoid defining variables in the module scope which ideally should contain only `H5Web` and `__version__`.

```suggestion
def _jupyter_labextension_paths():
    here = Path(__file__).parent.resolve()
    with (here / "labextension" / "package.json").open() as fid:
        data = json.load(fid)
    return [{"src": "labextension", "dest": data["name"]}]
```